### ISSUE TITLE
Splitter seperator role should be on gutter-handle not gutter

### ISSUE THREAD

**Issue by pdaechert** [primefaces/primevue#8381]
### Describe the bug

As state in https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/separator_role#focusable_separator the element with the role "seperator" should have the aria-valuenow and aria-orientation attributes. In Primevues Splitter the p-splitter-gutter element has the role and the p-splitter-gutter-handle is the element that is focusable and has the aria attributes.

### Reason for not contributing a PR

- [ ] Lack of time
- [x] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://stackblitz.com/edit/4wbpcjke?file=src%2FApp.vue

### Environment

The default implementation displayed on the primevue page has this issue.

### Vue version

^3.2.45

### PrimeVue version

4.5.4

### Steps to reproduce the behavior

1. check attributes of gutter element, see it has role=seperator
2. check attributes of gutter handle element, see it has aria attribute but not role attribute
3. try using tab to focus gutter, see that focus is on gutter handle => gutter handle should have role=seperator, gutter shouldn't

### Expected behavior

element that is focusable has role=seperator and aria-valuenow and aria-orientation attributes
